app/components/global/AppBar/ProfileMenu.js

```diff
@@ -141,6 +147,15 @@ class ProfileMenu extends React.Component {
                   <MenuHeading>{user.identities[0].name}</MenuHeading>
                 </MenuItem>
                 <MenuItem>
+                  <MenuItem>
+                    <MenuItemExternalLink
+                      data-test-id="manage-orcid"
+                      href="https://orcid.org/my-orcid"
```

Comment:
doesn't direct the user to the right place in `staging` however (https://sandbox.orcid.org/my-orcid)

Reply:
I will raise a ticket for making this a config item - thanks @giorgiosironi

Reply:
#1402